src/components/utils/test/auto_trace_test.cc

```diff
@@ -63,9 +62,7 @@ void Preconditions() {
 
 void InitLogger() {
   // Set enabled logs
-  profile::Profile::instance()->config_file_name("smartDeviceLink.ini");
-  profile::Profile::instance()->UpdateValues();
-  INIT_LOGGER("log4cxx.properties", true); //DEINIT_LOGGER will be called in test_main.cc
+  INIT_LOGGER("log4cxx.properties", true);  // DEINIT_LOGGER will be called in test_main.cc
```

Comment:
@Kozoriz 
Maybe I've missed that, but is .ini file name hardcoded in some other place?